src/components/GordonDialogBox/GordonDialogBox.module.scss

```
@@ -0,0 +1,18 @@
+@import '../../vars';
+
+.gc360_gordondialogbox {
+  display: grid;
+  grid-template-rows: auto auto auto;
+  overflow-y: auto;
+  &_title {
+    background-color: var(--mui-palette-primary-main);
+    color: $neutral-white;
+    text-align: center;
+  }
+  &_content :global(.MuiDialogTitle-root) {
+    padding: 8px 20px 8px 16px;
+  }
+  &_actions {
+    justify-content: space-evenly;
+  }
+}
```

Also, since this is a CSS module, we don't the very specific and long naming scheme.

It would be easier and clearer to flatten the CSS selectors and give them simpler, more clear names, e.g.
```suggestion
.dialog {
  display: grid;
  grid-template-rows: auto auto auto;
  overflow-y: auto;
}

.title {
    background-color: var(--mui-palette-primary-main);
    color: $neutral-white;
    text-align: center;
}

.content :global(.MuiDialogTitle-root) {
    padding: 8px 20px 8px 16px;
}

.actions {
    justify-content: space-evenly;
}
```

---

Yeah, I know you mentioned that would be good to change, now that we use modules.  I'll try to work on that at some point, but that'll be a bit of a project.